tests/DistortionTests.cpp

```diff
@@ -199,57 +199,58 @@ TEST(DawnFc, testZeroCoeffs) {
   ASSERT_DOUBLE_EQ(uy, 10.0);
 }
 
-TEST(KaguyaTc, testRemoveCoeffs) {
+TEST(KaguyaLism, testRemoveCoeffs) {
   csm::ImageCoord imagePt(1.0, 1.0);
 
   double ux, uy;
   double desiredPrecision = 0.0000001;
-  std::vector<double> distortionCoeffs = {1, 2, 3, 4,
-                                          1, 2, 3, 4};
+  std::vector<double> distortionCoeffs = {0.5, 1, 2, 3, 4,
+                                          0.5, 1, 2, 3, 4};
 
   removeDistortion(imagePt.samp, imagePt.line, ux, uy, distortionCoeffs,
-                  DistortionType::KAGUYATC, desiredPrecision);
+                  DistortionType::KAGUYALISM, desiredPrecision);
 
-  EXPECT_NEAR(ux, 1 + 1 + 2.828427125 + 6 + 11.313708499, 1e-8);
-  EXPECT_NEAR(uy, 1 + 1 + 2.828427125 + 6 + 11.313708499, 1e-8);
+  EXPECT_NEAR(ux, 1 + 1 + 2.828427125 + 6 + 11.313708499 + 0.5, 1e-8);
+  EXPECT_NEAR(uy, 1 + 1 + 2.828427125 + 6 + 11.313708499 + 0.5, 1e-8);
 }
 
 
-TEST(KaguyaTc, testCoeffs) {
+TEST(KaguyaLism, testCoeffs) {
   csm::ImageCoord imagePt(1.0, 1.0);
 
   double ux, uy, dx, dy;
   double desiredPrecision = 0.0000001;
   // Coeffs obtained from file TC1W2B0_01_05211N095E3380.img
-  std::vector<double> coeffs = {-0.0009649900000000001, 0.00098441, 8.5773e-06, -3.7438e-06,
-                                -0.0013796, 1.3502e-05, 2.7251e-06, -6.193800000000001e-06};
+  std::vector<double> coeffs = {-0.0725, -0.0009649900000000001, 0.00098441, 8.5773e-06, -3.7438e-06,
+                                0.0214, -0.0013796, 1.3502e-05, 2.7251e-06, -6.193800000000001e-06};
+
+  removeDistortion(imagePt.samp, imagePt.line, ux, uy, coeffs,
+                  DistortionType::KAGUYALISM, desiredPrecision);
+  applyDistortion(ux, uy, dx, dy, coeffs,
+                  DistortionType::KAGUYALISM, desiredPrecision);
```

Comment:
I swapped the order here so that I could hand calculate ux and uy. Before it did the iterative solution first which made hand calculating test data challenging.